packages/api-docs/src/layout.ts

```diff
@@ -47,6 +48,7 @@ const renderItem = (
             <div part="docs-column" class="column-type">
               <div part="docs-label">Type</div>
               <div part="docs-value">
+                ${isStatic ? 'static ' : ''}
```

Comment:
I think this would be better if
1. static properties were sorted out and displayed first or last
2. they had some visual indication that they are static other than the word `static`

Reply:
How does this look like?
![image](https://github.com/open-wc/api-viewer-element/assets/3748508/b62b6a4e-0b51-41a6-a570-c538adc21265)

Reply:
Yes something like that would be fine

Reply:
Colors changed to more neutral. Also, I would use this tag style later to display the "reflects" flag.

![image](https://github.com/open-wc/api-viewer-element/assets/3748508/86a85637-1547-4d77-a37b-46fc3f253a08)

Reply:
Sort function is added.